Frameworks/UIKit/UIFont.mm

```diff
@@ -309,15 +291,6 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     return [self retain];
 }
 
-- (void)_setName:(NSString*)name {
-    UNIMPLEMENTED();
-}
-
-- (uint32_t)_fontHandle {
-    UNIMPLEMENTED();
-    return StubReturn();
-}
-
 - (uint32_t)_sizingFontHandle {
```

Comment:
Currently 'used' in CGFont - I'll rip it out when I fix that

---
In reply to: [80597741](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80597741) [](ancestors = 80597741)